### ISSUE TITLE
A "My Watchlist" is not needed

### ISSUE THREAD

**Issue by scj643** [Sinap/plugin.video.funimation#5]
This feature isn't really needed since kodi has a favorites and any shows that would be on a "My Watchlist" would be put in that.

**Comment by afrase** [comment]
That's true but, I wanted it to function like Crunchyrolls queue. When you finish an episode it updates to the next one.

**Comment by johome6969** [comment]
anyone can help why incomplete with 1.1.2 ?
thanks

**Comment by afrase** [comment]
@johome6969 I'm not sure what you mean. I haven't really started working on a watch list yet.

**Comment by johome6969** [comment]
on all devices(fire tv--pc-pipo)... since 1.1.2 update the content of each show is only 2 or 3 episodes all others are missing..any help would be gr8 i love your addon quality is gr8..sorry to bother youjo 

```
 On Thursday, June 18, 2015 7:23 PM, Aaron Frase <notifications@github.com> wrote:
```

 @johome6969 I'm not sure what you mean. I haven't really started working on a watch list yet.—

**Comment by scj643** [comment]
Well I think it's an issue with the api for the premium content.

**Comment by scj643** [comment]
I modified the plugin.video.funimation/resources/lib/funimation.py and followed the comment that says you could watch the paid vidoes for free and changed 

``` python
self.user_type = 'FunimationUser'
```

to

``` python
self.user_type = 'FunimationSubscriptionUser'
```

**Comment by afrase** [comment]
@johome6969 It's because you haven't entered an account or don't have a paid account. As @scj643 pointed out, you can work around that.

**Comment by johome6969** [comment]
thanks !!!

**Comment by Protocol-X** [comment]
Favorites is nowhere near a replacement for your watchlist  imo.
